be/src/storage/schema_change.cpp

```diff
@@ -1277,6 +1283,17 @@ bool SchemaChangeWithSorting::process(RowsetReaderSharedPtr rowset_reader, Rowse
             LOG(WARNING) << "failed to allocate RowBlock.";
             result = false;
             goto SORTING_PROCESS_ERR;
+        } else {
+            // check allocator has enougth memory to do row-block sort
+            // in case of insufficient memory, row-block sort will fail which cause scheam change fail
```

Comment:
syntax error for schema change

Reply:
fix in commit https://github.com/StarRocks/starrocks/pull/225/commits/3a87767d271c123c35d91fdda5a98e13f6ecb0c2